src/UI/Badge.elm

```diff
@@ -178,6 +183,18 @@ successDark content =
     Badge { content = content, tone = ToneSuccess, brightness = Dark }
 
 
+{-| Replaces the tone of a badge with a new one.
+
+    prepareBadge model
+        |> (if model.requestStatus == Model.Failed then
+                Badge.withTone Badge.dangerLight
+
+            else
+                identity
+           )
+        |> Badge.renderElement renderConfig
+
+-}
```

Comment:
this example seems a little confusing, specially because we don't know what prepareBadge is doing at first. What about something like:
```elm
Badge.successLight "status"
  |> (if model.requestStatus == Model.Failed then
         Badge.withTone Badge.dangerLight
      else
         identity
      )
  |> Badge.renderElement renderConfig